haystack/nodes/prompt/providers.py

```diff
@@ -80,6 +73,10 @@ def known_providers() -> List[Type[PromptModelInvocationLayer]]:
     return [HFLocalInvocationLayer, OpenAIInvocationLayer, AzureOpenAIInvocationLayer]
 
 
+def instruction_following_models() -> List[str]:
+    return ["flan", "mt0", "bloomz", "davinci"]
```

Comment:
We could also include `iml` here for the [OPT-IML](https://huggingface.co/facebook/opt-iml-30b) models from Facebook.

Reply:
Sure let's gather the most current list of them as we move along. Feel free to add here @sjrl

Reply:
Sounds good! That's the only other one I could think of at the moment that's also on HuggingFace.